spec/authentication.yml

```diff
@@ -0,0 +1,823 @@
+          documentation: [Azure Authenticator](https://docs.conjur.org/Latest/en/Content/Operations/Services/azure_authn.htm) (`authn-azure`)."
+        operationId: "getAccessTokenViaAzure"
+        parameters:
+        - name: "service_id"
+          in: "path"
+          required: true
+          description: "URL-encoded authenticator service ID"
+          schema:
+            type: string
+          example: "prod/okta"
+        - name: "account"
+          in: "path"
+          required: true
+          description: "Organization account name"
+          schema:
+            $ref: 'openapi.yml#/components/schemas/AccountName'
+        - name: "login"
+          in: "path"
+          required: true
+          description: "URL-encoded login name. For users, it’s the user ID. For hosts, the login name is `host/<host-id>`"
+          schema:
+            $ref: '#/components/schemas/LoginName'
+        - name: "Accept-Encoding"
+          in: "header"
+          description: "Setting the Accept-Encoding header to base64 will return a pre-encoded access token"
+          schema:
+            type: "string"
+            enum:
+              - "application/json"
+              - "base64"
+            default: "application/json"
+        requestBody:
+          description: "Azure identity token"
+          required: true
+          content:
+            application/x-www-form-urlencoded:
+              schema:
+                $ref: 'openapi.yml#/components/schemas/AzureIdentityToken'
+
+        responses:
+          "200":
+            $ref: 'openapi.yml#/components/responses/AccessTokenGeneric'
+          "400":
+            $ref: 'openapi.yml#/components/responses/BadRequest'
+          "401":
+            $ref: 'openapi.yml#/components/responses/UnauthorizedError'
+          "404":
+            $ref: 'openapi.yml#/components/responses/ResourceNotFound'
+          "500":
+            $ref: 'openapi.yml#/components/responses/InternalServerError'
+
+
+    GCPAuthenticate:
+      post:
+        tags:
+        - "authentication"
+        summary: "Gets a short-lived access token for applications running in
+          Google Cloud Platform."
+        description: "Use the GCP Authenticator API to send an authentication
+          request from a Google Cloud service to Conjur.
+
+          For more information, see [the documentation](https://docs.conjur.org/Latest/en/Content/Operations/Services/cjr-gcp-authn.htm)."
+        operationId: "getAccessTokenViaGCP"
+        parameters:
+        - name: "account"
+          in: "path"
+          required: true
+          description: "Organization account name"
+          schema:
+            type: string
+          example: "dev"
+        - name: "Accept-Encoding"
+          in: "header"
+          description: "Setting the Accept-Encoding header to base64 will return a pre-encoded access token"
+          schema:
+            type: string
+            enum: [ "base64" ]
```

Comment:
Should this be updated to match the other `Accept-Encoding` variables?

Reply:
which is to say, should `application/json` be added as an option?